aasemble/django/apps/api/v1/serializers.py

```diff
@@ -109,10 +109,14 @@ def create(self, validated_data):
                 if 'tags' in validated_data:
                     tags_data = validated_data.pop('tags')
                     snapshot = mirrorsvc_models.Snapshot.objects.create(**validated_data)
```

Comment:
You could change the create call to: `create(visible_in_v1=(selff.view_prefix == 'v1'), **validated_data)`